src/main/java/svc/security/HashUtil.java

```diff
@@ -0,0 +1,59 @@
+package svc.security;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
+public class HashUtil{
+	@Value("${stlcourts.security.municipalitySalt}")
+	String municipalitySalt;
+	@Value("${stlcourts.security.judgeSalt}")
+	String judgeSalt;
+	@Value("${stlcourts.security.courtSalt}")
+	String courtSalt;
+	
+	public Hashids municipalityHashids;
+	public Hashids judgeHashids;
+	public Hashids courtHashids;
+	
+	private void initializeSalts(){
```

Comment:
OK - let's re-evaluate later. It's pretty inefficient to build these things new over and over with the same salt